docs/whatsnew/2.0.rst

```diff
@@ -249,26 +236,88 @@ to atomic mass unit (amu). For example::
     >>> y.to(u.g/u.mol, equivalencies=u.molar_mass_amu())
     <Quantity 1.0 g / mol>
 
-.. _whatsnew-2.0-models-units:
+.. _whatsnew-2.0-ascii-ecsv-mixins:
 
-New unit support for most models
-================================
+Store astropy core object types in ASCII ECSV table file
+========================================================
 
-Most Astropy models now can handle inputs with units, and produce the
-appropriate outputs with units as well. Some models cannot support this due
-to their definitions (e.g., Legendre, Hermite, etc), while some will have
-this capability added in a future release. Example usage::
+It is now possible to store the following :ref:`mixin column
+<mixin_columns>` types in an ASCII :ref:`ECSV
+<ecsv_format>` table file.  The table file can then be read back
+into astropy with no loss of object data or attributes.
 
-    >>> from astropy import units as u
-    >>> from astropy.modeling.models import Gaussian1D
-    >>> g = Gaussian1D(amplitude=1*u.J, mean=1*u.m, stddev=0.1*u.m)
-    >>> g([3, 4, 5.5] * u.cm)
-    <Quantity [  3.70353198e-21,  9.72098502e-21,  4.05703276e-20] J>
+- `astropy.time.Time`
```

Comment:
this list looks rather odd in the rendered version. Wouldn't it be enough just to list the short names in a list after each other?